applications/nrf_desktop/doc/board.rst

```diff
@@ -0,0 +1,39 @@
+.. _board:
+
+Board module
+############
+
+The ``board`` module is used to assure proper state of GPIO pins which are connected to electrical components and not configured by other modules in the application.
```

Comment:
```suggestion
The ``board`` module is used to ensure the proper state of GPIO pins that are connected to electrical components and not configured by other modules in the application.
```
How does it ensure this? Mention this briefly here.

Reply:
This is assured by setting state of GPIO pins, I will add this information here.